### PR TITLE
[dev] No PBI - Spread styles object for each columns on Data Card.

### PR DESCRIPTION
**Pull request by alejosod**

### CODE REVIEW

**Review by cbrewer1689 — APPROVED** [review]
Looks good.

File: src/dataDisplay/dataCards/dataCardColumn.jsx [hunk 1]
```diff
@@ -37,6 +43,7 @@ function DataCardColumn(props) {
                 style={{
                     marginBottom: column.width ? '11px' : null,
                     width: column.width,
+                    ...column.style,
```
Comment (groberts314) [inline src/dataDisplay/dataCards/dataCardColumn.jsx:46]: Do we need to defensively check for `style`?
Maybe `...(style ?? {})` or something.

**Review by groberts314 — CHANGES_REQUESTED** [review]
One concern about being defensive if `column.style` is nullish.  Otherwise looks good.

File: src/dataDisplay/dataCards/dataCardColumn.jsx (hunk 1 above)
Comment (cbrewer1689) [reply src/dataDisplay/dataCards/dataCardColumn.jsx:46]: Yeah, I thin we need to be defensive. Good catch.

Comment (cbrewer1689) [reply src/dataDisplay/dataCards/dataCardColumn.jsx:46]: ...

Comment (groberts314) [reply src/dataDisplay/dataCards/dataCardColumn.jsx]: Looks better now.  Resolving.

**Review by groberts314 — APPROVED** [review]

**Review by War777 — APPROVED** [review]
Looks good to me

**Pull request merged**